Cannot build with gcc-bridge on macOS 10.12.5

I cannot build recent versions of the mode on macOS 10.12.5 (16F73) -- the step to download gcc-bridge fails.

1.  checked out aed8ba6
2.  built with `./scripts/generate-ant-file.sh; ant install`

See the log below for details on gcc-bridge. Strange that it is passing Travis-CI correctly. Is there a new OS-level dependency?


```
BUILD FAILED
/Users/jeremydouglass/git/Processing.R/build.xml:43: Unable to resolve artifact: Missing:
----------
1) org.renjin:gcc-bridge-plugin:jar:apple-x86_64:0.8.2411

  Try downloading the file manually from the project website.

  Then, install it using the command: 
      mvn install:install-file -DgroupId=org.renjin -DartifactId=gcc-bridge-plugin -Dversion=0.8.2411 -Dclassifier=apple-x86_64 -Dpackaging=jar -Dfile=/path/to/file

  Alternatively, if you host your own repository you can deploy the file there: 
      mvn deploy:deploy-file -DgroupId=org.renjin -DartifactId=gcc-bridge-plugin -Dversion=0.8.2411 -Dclassifier=apple-x86_64 -Dpackaging=jar -Dfile=/path/to/file -Durl=[url] -DrepositoryId=[id]

  Path to dependency: 
  	1) org.apache.maven:super-pom:pom:2.0
  	2) org.renjin:renjin-cli:jar:0.8.2411
  	3) org.renjin:renjin-gnur-compiler:jar:0.8.2411
  	4) org.renjin:gcc-bridge-compiler:jar:0.8.2411
  	5) org.renjin:gcc-bridge-plugin:jar:apple-x86_64:0.8.2411

----------
1 required artifact is missing.

for artifact: 
  org.apache.maven:super-pom:pom:2.0

from the specified remote repositories:
  central (http://repo1.maven.org/maven2),
  remote.repository (https://nexus.bedatadriven.com/content/groups/public/)

```

--- a/build.xml.template
+++ b/build.xml.template
@@ -64,12 +64,15 @@
             <os family="windows" />
         </condition>
 
-        <condition property="isLinux" value="true">
-            <os family="unix" />
+        <condition property="isMacOS" value="true">
+            <os family="mac" />
         </condition>
 
-        <condition property="isLinux" value="false">
-            <os family="mac" />
+        <condition property="isLinux" value="true">
+            <and>
+                <isfalse value="${isMacOS}" />
+                <os family="unix" />
+            </and>
         </condition>
     </target>
 

--- a/build.xml.template
+++ b/build.xml.template
@@ -67,6 +67,10 @@
         <condition property="isLinux" value="true">
             <os family="unix" />
         </condition>
+
+        <condition property="isLinux" value="false">
+            <os family="mac" />
+        </condition>
     </target>
 
     <target name="dep" depends="checkos" if="isLinux">

